spec/lib/metrics_remove_blog_post_count_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'rake'
```

Comment:
Is this actually needed?

Reply:
It is, because of the rake utils I'm using in the spec